shared/src/test/diff/nu/SimpleUCS.mls

```diff
@@ -234,49 +208,48 @@ fun f(x, a, b) =
 //│ f: ((A with {value: 'value}) | (B with {value: 'value}) | ~A & ~B, anything, anything,) -> (0 | 'value)
 //│  = [Function: f14]
 
-fun f(x, y) =
+fun f(x, y, b) =
   if x is
     Some(xv) and y
       is Some(yv) then "bruh"
```

Comment:
Curious to see what happens if you make it
```suggestion
      is Some(yv) then yv
```
(please add this as an additional test)

Reply:
It's added.